src/targets/browser/setupApp.jsx

```diff
@@ -14,6 +15,8 @@ const setupApp = memoize(() => {
   const { lang, appName } = getValues(JSON.parse(root.dataset.cozy))
   const polyglot = initTranslation(lang, lang => require(`locales/${lang}`))
   const client = getClient()
+  client.registerPlugin(RealtimePlugin)
```

Comment:
Only two lines of code to add realtime sync? What kind of sorcery is this? 😱

Reply:
You can create an entry for a future PDC session. I will explain how it works :)